html-forms.tar/html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <p>Email</p>
+    <input type="text" id="comment" name="comment" value="" />
+    <p>Why is coding cool?</p>
+    <select id="reason" name="userchoice">
+      <option value="It is fun">It is fun</option>
+      <option value="It improves your logical skill">It improves your logical skill</option>
+      <option value="It helps you be more adaptable to technology">It helps you be more adaptable to technology</option>
+    </select>
+    <p>Favorite language (choose one)</p>
+    <input type="radio" name="language" id="language_1" value="HTML" />
+      <label for="language_1">HTML</label>
+      <input type="radio" name="language" id="language_2" value="CSS" />
+      <label for="language_2">CSS</label>
+      <input type="radio" name="language" id="language_3" value="JavaScript" />
+      <label for="language_3">JavaScript</label>
+      <p>Would you like to sign up for more information?</p>
+      <input type="checkbox" id="check_yes" name="user_check" value="Sign me up!" />
+    <label for="user_check">Sign me up!</label>
```

Comment:
the `id` of the `input` tag and the `for` attribute of the `label` tag need to match